frontend/public/locales/en/cluster.json

```diff
@@ -75,16 +75,19 @@
     "submariner.install.form.config.doc": "How do I configure Submariner?",
     "submariner.install.form.credential.secret": "Provider credential secret",
     "submariner.install.form.nattport": "IPSec NAT-T port",
+    "submariner.install.form.nattenable": "NATT Enable",
     "submariner.install.form.cabledriver": "Cable driver",
     "submariner.install.form.gateways": "Gateway count",
     "submariner.install.form.instancetype": "Instance type",
     "submariner.install.form.clusters.placeholder": "Select clusters",
     "submariner.install.form.port.placeholder": "Enter port number",
+    "submariner.install.form.nattenable.placeholder": "Select natt enabled",
     "submariner.install.form.cabledriver.placeholder": "Select cable driver",
     "submariner.install.form.gateways.placeholder": "Enter gateway count",
     "submariner.install.form.instancetype.placeholder": "Select instance type",
     "submariner.install.form.credential.secret.labelHelp": "The reference to the Secret in the cluster namespace that contains the provider credentials. The credentials will be used to prepare the Submariner environment on the provider.",
     "submariner.install.form.nattport.labelHelp": "The Submariner creates the IPsec tunnel between the clusters.  This port is used for IPsec NAT traversal. (default 4500)",
+    "submariner.install.form.nattenable.labelHelp": "The Submariner can use NAT Traversal for IPsec tunnel between clusters. Set this to true to enable and false to disable NAT Traversal (default true)",
```

Comment:
```suggestion
    "submariner.install.form.nattenable.labelHelp": "Submariner can use NAT Traversal for IPsec tunnels between clusters. Set this to true to enable NAT Traversal and false otherwise (default true)",
```